src/test_application/OSPMagnum.cpp

```diff
@@ -181,11 +182,11 @@ void testapp::config_controls(OSPMagnum& rOspApp)
 
             //Add user input
             rUserInput.config_register_control(k, holdable, controls);
-
         }
     }
-    catch (std::out_of_range oor) {
-        std::cout << oor.what() << std::endl;
+    catch (const std::exception& exception) 
+    {
+        std::cout << exception.what() << std::endl;
```

Comment:
I'm not sure I agree with just catching the exception and doing nothing. The program is probably in a bad state if an exc eption gets to this level.

Just let the exception go to the top level and crash the program